diff --git a/ImageTools.py b/ImageTools.py
index d909d83..8658c2d 100644
--- a/ImageTools.py
+++ b/ImageTools.py
@@ -11,6 +11,7 @@ from tools.rawimageeditor.RawImageEditor import RawImageEditor
 from tools.video_compare.videocompare import VideoCompare
 from tools.pqtools_to_code.pqtools_to_code import PQtoolsToCode
 from tools.yuv_viewer.yuv_viewer import YUVViewer
+from tools.batch_rename.batch_rename import BatchRenameTool
 from components.check_update import CheckUpdate, simple_check_is_need_update
 import components.logconfig as log
 from logging import info
@@ -24,6 +25,7 @@ class ImageTools(MainWindow):
             "FieldDepthWindow": [self.ui.field_depth_tool, FieldDepthWindow],
             "ShakeTestTool": [self.ui.shake_tool, ShakeTestTool],
             "ImageEditor": [self.ui.imageeditor, ImageEditor],
+            "BatchRenameTool": [self.ui.batch_rename, BatchRenameTool],
             "RawImageEditor": [self.ui.rawimageeditor, RawImageEditor],
             "VideoCompare": [self.ui.video_compare, VideoCompare],
             "HelpDoc": [self.ui.userguide, HelpDoc],
@@ -35,7 +37,6 @@ class ImageTools(MainWindow):
         self.ui.clearcache.triggered.connect(self.clear_cache)
         self.ui.checkupdate.triggered.connect(self.add_checkupdate_window)
         for (key, value) in self.subwindow_function.items():
-            # 注意，这个lambda表达式必须要先赋值才能使用，否则connect的永远是最后的一个类
             value[0].triggered.connect(
                 lambda win_name=key, win_object=value[1]: self.add_sub_window(win_name, win_object))
 
diff --git a/components/ui/mainwindow.py b/components/ui/mainwindow.py
index 92b6a5b..c81ea71 100644
--- a/components/ui/mainwindow.py
+++ b/components/ui/mainwindow.py
@@ -1,13 +1,3 @@
-# -*- coding: utf-8 -*-
-
-################################################################################
-## Form generated from reading UI file 'mainwindow.ui'
-##
-## Created by: Qt User Interface Compiler version 5.15.2
-##
-## WARNING! All changes made in this file will be lost when recompiling UI file!
-################################################################################
-
 from PySide2.QtCore import *
 from PySide2.QtGui import *
 from PySide2.QtWidgets import *
@@ -26,6 +16,8 @@ class Ui_MainWindow(object):
         self.shake_tool.setObjectName(u"shake_tool")
         self.imageeditor = QAction(MainWindow)
         self.imageeditor.setObjectName(u"imageeditor")
+        self.batch_rename = QAction(MainWindow)
+        self.batch_rename.setObjectName(u"batch_rename")
         self.about = QAction(MainWindow)
         self.about.setObjectName(u"about")
         self.rawimageeditor = QAction(MainWindow)
@@ -80,6 +72,7 @@ class Ui_MainWindow(object):
         self.menubar.addAction(self.menu.menuAction())
         self.menu_2.addAction(self.shake_tool)
         self.menu_2.addAction(self.imageeditor)
+        self.menu_2.addAction(self.batch_rename)
         self.menu_2.addAction(self.rawimageeditor)
         self.menu_2.addAction(self.video_compare)
         self.menu_2.addAction(self.yuv_viewer)
@@ -92,24 +85,22 @@ class Ui_MainWindow(object):
         self.retranslateUi(MainWindow)
 
         QMetaObject.connectSlotsByName(MainWindow)
-    # setupUi
 
     def retranslateUi(self, MainWindow):
         MainWindow.setWindowTitle(QCoreApplication.translate("MainWindow", u"ImageTools", None))
-        self.shake_tool.setText(QCoreApplication.translate("MainWindow", u"\u6296\u52a8\u6d4b\u8bd5\u5de5\u5177", None))
-        self.imageeditor.setText(QCoreApplication.translate("MainWindow", u"\u56fe\u7247\u5206\u6790\u5de5\u5177", None))
-        self.about.setText(QCoreApplication.translate("MainWindow", u"\u5173\u4e8e", None))
-        self.rawimageeditor.setText(QCoreApplication.translate("MainWindow", u"raw\u56fe\u5206\u6790\u5de5\u5177", None))
-        self.video_compare.setText(QCoreApplication.translate("MainWindow", u"\u89c6\u9891\u5bf9\u6bd4\u5de5\u5177", None))
-        self.pqtools2code.setText(QCoreApplication.translate("MainWindow", u"PQtools\u8f6c\u4ee3\u7801", None))
-        self.field_depth_tool.setText(QCoreApplication.translate("MainWindow", u"\u955c\u5934\u8ba1\u7b97\u5668", None))
-        self.af_calc_tool.setText(QCoreApplication.translate("MainWindow", u"\u955c\u5934\u66f2\u7ebf\u8ba1\u7b97\u5de5\u5177", None))
-        self.userguide.setText(QCoreApplication.translate("MainWindow", u"\u7528\u6237\u624b\u518c", None))
-        self.clearcache.setText(QCoreApplication.translate("MainWindow", u"\u6e05\u7406\u7f13\u5b58", None))
-        self.checkupdate.setText(QCoreApplication.translate("MainWindow", u"\u68c0\u67e5\u66f4\u65b0", None))
-        self.yuv_viewer.setText(QCoreApplication.translate("MainWindow", u"YUV\u67e5\u770b\u5de5\u5177", None))
-        self.menu_2.setTitle(QCoreApplication.translate("MainWindow", u"\u5de5\u5177", None))
+        self.shake_tool.setText(QCoreApplication.translate("MainWindow", u"抖动测试工具", None))
+        self.imageeditor.setText(QCoreApplication.translate("MainWindow", u"图片分析工具", None))
+        self.batch_rename.setText(QCoreApplication.translate("MainWindow", u"图片批量重命名", None))
+        self.about.setText(QCoreApplication.translate("MainWindow", u"关于", None))
+        self.rawimageeditor.setText(QCoreApplication.translate("MainWindow", u"raw图分析工具", None))
+        self.video_compare.setText(QCoreApplication.translate("MainWindow", u"视频对比工具", None))
+        self.pqtools2code.setText(QCoreApplication.translate("MainWindow", u"PQtools转代码", None))
+        self.field_depth_tool.setText(QCoreApplication.translate("MainWindow", u"镜头计算器", None))
+        self.af_calc_tool.setText(QCoreApplication.translate("MainWindow", u"镜头曲线计算工具", None))
+        self.userguide.setText(QCoreApplication.translate("MainWindow", u"用户手册", None))
+        self.clearcache.setText(QCoreApplication.translate("MainWindow", u"清理缓存", None))
+        self.checkupdate.setText(QCoreApplication.translate("MainWindow", u"检查更新", None))
+        self.yuv_viewer.setText(QCoreApplication.translate("MainWindow", u"YUV查看工具", None))
+        self.menu_2.setTitle(QCoreApplication.translate("MainWindow", u"工具", None))
         self.menuISP.setTitle(QCoreApplication.translate("MainWindow", u"ISP", None))
-        self.menu.setTitle(QCoreApplication.translate("MainWindow", u"\u5173\u4e8e", None))
-    # retranslateUi
-
+        self.menu.setTitle(QCoreApplication.translate("MainWindow", u"关于", None))
diff --git a/components/ui/mainwindow.ui b/components/ui/mainwindow.ui
index 8511408..1807b40 100644
--- a/components/ui/mainwindow.ui
+++ b/components/ui/mainwindow.ui
@@ -67,6 +67,7 @@
     </property>
     <addaction name="shake_tool"/>
     <addaction name="imageeditor"/>
+    <addaction name="batch_rename"/>
     <addaction name="rawimageeditor"/>
     <addaction name="video_compare"/>
     <addaction name="yuv_viewer"/>
@@ -100,6 +101,11 @@
     <string>图片分析工具</string>
    </property>
   </action>
+  <action name="batch_rename">
+   <property name="text">
+    <string>图片批量重命名</string>
+   </property>
+  </action>
   <action name="about">
    <property name="text">
     <string>关于</string>
